assets/stylesheets/common/common.scss

```diff
@@ -1303,11 +1308,18 @@ body.composer-open .topic-chat-float-container {
   }
 
   .chat-uploads {
+    margin-top: 0.25em;
+
     .chat-img-upload,
     .chat-other-upload {
       margin-bottom: 1em;
     }
   }
+
+  a.onebox img {
```

Comment:
Hmm, why do need to unset height/width? (There's probably a good reason, but it can be a sign of a problem elsewhere in the stylesheet.)

Reply:
It's because of the way the oneboxes are created now. If it isn't unset, the images look like that:

<img width="815" alt="Screenshot 2022-01-12 at 10 30 08 PM" src="https://user-images.githubusercontent.com/1555215/149159355-62d6e67c-2f60-488a-be99-3d2798946d46.png">

T_T